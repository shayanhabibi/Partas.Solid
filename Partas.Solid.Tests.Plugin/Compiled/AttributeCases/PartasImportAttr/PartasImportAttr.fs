module Partas.Solid.Tests.AttributeCases.PartasProxyImportAttr

open Partas.Solid
open Partas.Solid.Router
open Fable.Core

// todo-replace with a better test
[<Erase>]
type PartasImportAttr() =
    interface VoidNode

    [<SolidTypeComponentAttribute>]
    member props.constructor = Router () { Route () }
