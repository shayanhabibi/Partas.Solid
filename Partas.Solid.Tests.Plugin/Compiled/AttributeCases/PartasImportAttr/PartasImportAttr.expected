import { splitProps } from "solid-js";
import { Route, Router } from "@solidjs/router";

export function PartasImportAttr(props) {
    const [PARTAS_LOCAL, PARTAS_OTHERS] = splitProps(props, []);
    return <Router>
        <Route />
    </Router>;
}

